srcs/Cgi.cpp

```diff
@@ -114,10 +113,10 @@ namespace Webserver
 		// std::string	completeCgiTarget = prependRoot(_host.getRoot(), _request.getTarget());
 		std::string	queryString = createQueryString();
 		const char* env[] = {queryString.c_str(), NULL};
-		const char* argv[] = {"python3", _uri.c_str(), NULL};
+		const char* argv[] = {"python3", _uri.getTarget().c_str(), NULL};
 
-		if (access(_uri.c_str(), F_OK ) == SYSTEM_ERR || _cgiExecutable == "")
-			exit(2);
+		if (!_uri.entryExists() || _cgiExecutable == "")
+			throw InvalidRequestException(HttpStatusCodes::INTERNAL_ERROR);
```

Comment:
this is in the child process, correct? It won't send any response from this exception